mlir/include/mlir/Dialect/Transform/IR/TransformDialect.h

```diff
@@ -315,6 +315,62 @@ class BuildOnly : public DerivedTy {
   BuildOnly() : DerivedTy(/*buildOnly=*/true) {}
 };
 
+namespace detail {
+
+/// Expands the given list of `paths` to a list of `.mlir` files.
+///
+/// Each entry in `paths` may either be a regular file, in which case it ends up
+/// in the result list, or a directory, in which case all (regular) `.mlir`
+/// files in that directory are added. Any other file types lead to a failure.
+LogicalResult expandPathsToMLIRFiles(ArrayRef<std::string> &paths,
```

Comment:
Why take a reference to ArrayRef, which itself is a reference as the name indicates?

(I see the code is moved from elsewhere, but IIRC it's yours anyway).

Reply:
No good reason. I probably changed the type from something else and overlooked the `&` when I changed it to `ArrayRef`. Fixed in new PR.